static/js/email_suggestion.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
+const email = jQuery('#email');
+const hint = jQuery('#hint');
```

Comment:
We'll want more precise selectors for the target fields; otherwise this might accidentally run on parts of pages it isn't meant to.